### ISSUE TITLE
metric of nerf-synthetic/ficus

### ISSUE THREAD

**Issue by DHPark98** [cwchenwang/NeRF-SR#12]
Hello, thank you for your work and code. 
For Blender x4(200x200, ss & w/o refine), I ran the "train_blender_downX.sh".
For ficus dataset of NeRF-synthetic, psnr that I got was 22.69, which is quite lower that I expected.
So could you give your psnr number of ficus?

Of course, I changed the hyperparameters :
H=800, W=800, and downscale=4.

I read events file for psnr :
logs/nerf-sr/blender-down4-ficus-800x800-ni64-dp-ds4/fine_psnr_ori_val_full -> 22.69

And after I render the test set("test_blender_downX.sh"), I also calculated the psnr between rendered views and test GT manually:
22.70

**Comment by cwchenwang** [comment]
Hi, the psnr for ficus is 26.635. I htink the fine_psnr_ori_val_full is only evaluated on one view, not the full scene.

**Comment by DHPark98** [comment]
I trained nerf-sr w/ ficus dataset once again, and rendered test views.
Unfortunately, PSNR didn't increase : 22.73
Could you check my bash file below..?


dataset="ficus"
epoch=20
W=800
H=800
accelerator="dp"
downscale=4
N_importance=64
batch_size=1024
CUDA_VISIBLE_DEVICE=0 python train_blender.py --name blender-down${downscale}-$dataset-${H}x${W}-ni${N_importance}-${accelerator}-ds${downscale} --accelerator $accelerator \
    --dataset_mode blender_downX --dataset_root dataset/nerf_synthetic/${dataset} --val_epoch_split test \
    --checkpoints_dir ./checkpoints/nerf-sr --summary_dir ./logs/nerf-sr \
    --img_wh $W $H --batch_size $batch_size \
    --n_epochs ${epoch} --n_epochs_decay 10 \
    --print_freq 100 --vis_freq 1000 --val_freq 1000 --val_epoch_freq 20 --vis_epoch_freq 20 --save_epoch_freq 5 \
    --model nerf_downX --N_coarse 64 --N_importance $N_importance \
    --lr_policy exp --sigma_activation relu --lr 5e-4 --lr_final 5e-6 \
    --downscale ${downscale}

**Comment by cwchenwang** [comment]
How did you get the PSNR? On all images?

**Comment by DHPark98** [comment]
yes, average PSNR of 200 test images.

**Comment by cwchenwang** [comment]
That's weird. The training script looks ok to me, I can try to find my previous checkpoints, but it may take some time.
